packages/x-charts/src/LineChart/LineElement.tsx

```diff
@@ -87,28 +91,46 @@ LineElementPath.propTypes = {
   ]),
 } as any;
 
-export type LineElementProps = Omit<LineElementOwnerState, 'isFaded' | 'isHighlighted'> &
-  React.ComponentPropsWithoutRef<'path'> & {
-    highlightScope?: Partial<HighlightScope>;
-    /**
-     * The props used for each component slot.
-     * @default {}
-     */
-    slotProps?: {
-      line?: SlotComponentProps<'path', {}, LineElementOwnerState>;
-    };
-    /**
-     * Overridable component slots.
-     * @default {}
-     */
-    slots?: {
-      /**
-       * The component that renders the root.
-       * @default LineElementPath
-       */
-      line?: React.ElementType;
-    };
-  };
+export interface LineElementSlots {
+  /**
+   * The component that renders the root.
```

Comment:
Nitpick: WDYT, would it have made sense to call it `root` as we usually do with other components (packages)? 🤔

Reply:
I did the following:

```diff
- The component that renders the root
+ The component that renders the line
```

Because this slots is extended to the `<LinePLot />` and to the `<LineChart />` so the definition was incorrect